### PR TITLE
added pytest

### PR DESCRIPTION
**Pull request by lucien386**

### CODE REVIEW

**Comment by lucien386** [comment]
```
ImportError while loading conftest '/home/travis/build/lucien386/MusicXML2ABCNotationWebApp/tests/conftest.py'.
tests/conftest.py:2: in <module>
    from converter import create_app
E   ModuleNotFoundError: No module named 'converter'
The command "pytest" exited with 4.
```

**Comment by lucien386** [comment]
The problem appears as well when I am running locally. but I use `python3 -m pytest /tests` instead and it works. Im looking at this problem rn

File: tests/test_homepage.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+def test_home_page(test_client):
+    """
+    GIVEN a Flask application
+    WHEN the '/' page is requested (GET)
+    THEN check the response is valid
+    """
+    response = test_client.get('/')
+    assert response.status_code == 200
+    assert b"MusicXML to ABC Notation Converter" in response.data
```
Comment (xicheng87) [inline tests/test_homepage.py:9]: Why is there a "b" in front of the string here?

File: tests/test_submit.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+import io
+
+def test_submit_IncorrectMusicXml(test_client):
+    #Test can upload and detect incorrect text.
+    data = {"content": "abcde"}
+    data = {key: str(value) for key, value in data.items()}
+    response = test_client.post(
+        '/convert_result/submission', data=data, follow_redirects=True
+    )
+    assert b"Result" in response.data
```
Comment (xicheng87) [inline tests/test_submit.py:11]: Ditto.

File: tests/test_homepage.py (hunk 1 above)
Comment (lucien386) [reply tests/test_homepage.py:9]: to note that it is a binary string, i could get rid of that

Comment (xicheng87) [reply tests/test_homepage.py:9]: Thanks for the clarification. Not a big deal for now. Let's first get the build to pass

**Comment by lucien386** [comment]
it turns out that i have to do `pip install -e .`

**Pull request merged**